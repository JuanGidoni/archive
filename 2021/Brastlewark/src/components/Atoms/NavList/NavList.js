import NavLink from '../NavLink';

const NavList = ({ text, to }) => {
 return (
  <li className="nav-item">
   <NavLink to={to}>{text}</NavLink>
  </li>
 )
}

export default NavList
