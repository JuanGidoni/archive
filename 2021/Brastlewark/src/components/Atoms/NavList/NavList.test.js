import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import NavList from './NavList';

describe('<NavList />', () => {
 it('renders correctly and match snapshot', () => {
  const tree = renderer
   .create(<Router><NavList text="Example" to="/" /></Router>)
   .toJSON();
  expect(tree).toMatchSnapshot();
 });
});
