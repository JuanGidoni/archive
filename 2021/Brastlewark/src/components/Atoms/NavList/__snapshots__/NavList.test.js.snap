// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavList /> renders correctly and match snapshot 1`] = `
<li
  className="nav-item"
>
  <a
    className="nav-link "
    href="/"
    onClick={[Function]}
  >
    Example
  </a>
</li>
`;
